docs/docs/theme_management.md

```diff
@@ -39,6 +39,9 @@ Unlike templates you can have more than one static_path you wish to include. You
 ```python
 from render_engine import Site
 
+class MySite(Site):
+    static_path = "my_static"
+    
```

Comment:
```suggestion
    ```
Another option is to apply the static path after initialization as the paths are not merged until `Site.render()` is called.
```

Reply:
You will likely need to make this change in your code as I can't escape the code blocks in the suggestion.